base/linalg/dense.jl

```diff
@@ -551,40 +551,26 @@ julia> logm(A)
  0.0  1.0
 ```
 """
-function logm(A::StridedMatrix{T}) where T
+function logm{T}(A::StridedMatrix{T})
```

Comment:
For future reference, is there something wrong with this, or is it just the preferred syntax?